.gitignore

```diff
@@ -36,3 +43,4 @@ common.pyc
 /gh-pages/_site
 /*compiler*.jar
 /local_blockly_compressed_vertical.js
+/chromedriver
```

Comment:
This is one minor addition to clean-up an ergonomics issue. Happy to split this out if you prefer @picklesrus @rschamp.